go/adbc/pkg/flightsql/driver.go

```diff
@@ -184,7 +185,7 @@ func FlightSQLDatabaseRelease(db *C.struct_AdbcDatabase, err *C.struct_AdbcError
 	C.free(unsafe.Pointer(db.private_data))
 	db.private_data = nil
 	h.Delete()
-
+	runtime.GC()
```

Comment:
Were these added to the template as well?

Reply:
Also, can we document the "why" of why we have to GC here?

Reply:
added